Makefile

```diff
@@ -0,0 +1,26 @@
+-include local.mk
```

Comment:
Do we need make file, if we use github actions. As I remember there were some issues with make files previously.